app/views/katello/api/v2/subscriptions/base.json.rabl

```diff
@@ -13,3 +13,12 @@ attributes :sockets, :cores, :ram
 attributes :instance_multiplier, :stacking_id, :multi_entitlement
 attributes :type
 attributes :name => :product_name
+attributes :unmapped_guest
```

Comment:
What does this attribute actually mean? I looked through the code, and it appears from Candlepin it comes in as `unmapped_guests_only` which has a different meaning than `unmapped_guest` to me.